apps/flux-system/ptl-intsvc/base/kustomization.yaml

```diff
@@ -2,11 +2,16 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: flux-system
 resources:
-  - ../../base
-  - ../../base/image-automation-components.yaml
+  - ./gotk-components.yaml
```

Comment:
or move [this](https://github.com/hmcts/cnp-flux-config/blob/master/apps/flux-system/base/kustomization.yaml#L4) to all envs? ? Just conscious we are adding many files here seperately which could get missed when adding new ones in future ?

Reply:
yeah i see what you're saying here. I think you're right and this may be a better implementation. I'll switch to this method.